### PR TITLE
Upgrade/operator sdk

### PR DESCRIPTION
**Pull request by plaffitt**
- Upgrade operator-sdk from 0.8.1 to 1.5.2 🎉
- Upgrade golang from <1.13 (before go modules are the default) to golang 1.17 🎉
- Upgrade crdVersion from v1alpha1 to v1 🎉
- Start building the operator binary inside the Dockerfile with multi-stage build instead of doing it outside before copying it 🎉
- Fix missing required namespace in ClusterRoleBinding 🛠️

### CODE REVIEW

File: .travis.yml
Comment (niqdev) [inline .travis.yml:6]: once this gets merged in the main repo i'll replace travis with github-actions 😉

**Comment by niqdev** [comment]
Hey, is there any update on this? I'd like to test and use the latest version of the operator in a cluster, and the work you guys have done is great!

If you have any concerns about the project maintenance, I'd be more than happy to make you codeowners or collaborators of the project, so that should give you enough freedom to upgrade it and have control over it - I'm looking for help basically 😛 
I can take care of fixing the automatic release with GitHub action on DockerHub if this gets merged on the baseline.

Please let me know what you think, thanks!

**Review by donch — APPROVED** [review]

**Comment by plaffitt** [comment]
@niqdev I'm going to publish a pull-request to upstream

**Pull request merged**
